index.md

```diff
@@ -226,13 +226,15 @@ for the latter instance, in which case it's obviously a minor level increment.
 
 #### Special note for JVM or similar environments
 
-In such environments where only one version of a library could 
+I suggest to add something like "[...] If any potential client environment allows only one version of a library be present in running system, [...] user library MUST be [...]"
```

Comment:
I guess line 229 (starting with "I suggest ...") was added in your last commit for mistake, it should be removed.

Reply:
Yeah, I'm sorry for the inconvenience.